django_reference_app/tests/test_model.py

```diff
@@ -0,0 +1,11 @@
+from django import test
+from django_reference_app.models import Person
+
+class PersonTest(test.TestCase):
+  def create_person(self, name="test", job="only a test"):
+    return Person.objects.create(name=name, job=job)
+
+  def test_person_creation(self):
+    w = self.create_person()
+      self.assertTrue(isinstance(w, Person))
```

Comment:
I'd drop this test - we only really need one test, and the next `assertEqual` test is fine - plus the use of `isinstance` is often frowned upon in Python due to its preference for [duck typing](https://www.quora.com/What-is-Duck-typing-in-Python).

Reply:
I misread this, myself at first.  I'd add that individual unit tests should only contain a single assertion per test.